src/Orleans.Streaming.GCP/Orleans.Streaming.GCP.csproj

```diff
@@ -11,7 +11,7 @@
 
   <PropertyGroup>
     <!-- Cannot release a stable version of this package that depends on a prerelease version of Google.Cloud.PubSub.V1. -->
-    <IsPackable>false</IsPackable>
+    <IsPackable>true</IsPackable>
```

Comment:
Maybe we should rename this package to `Microsoft.Orleans.Streaming.GoogleCloudPubSub` to match the dependency package name (i.e, Google.Cloud.PubSub.V1)